source/features/remove-label-button.tsx

```diff
@@ -0,0 +1,86 @@
+import './remove-label-button.css';
+import React from 'dom-chef';
+import XIcon from 'octicon/x.svg';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+import {observe} from 'selector-observer';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+
+import {getRepoURL, getConversationNumber} from '../github-helpers';
+
+const canNotEditLabels = oneTime((): boolean => !select.exists('.sidebar-labels .octicon-gear'));
+
+function updateSidebar() {
+	select('#partial-discussion-sidebar')!.dispatchEvent(new CustomEvent('socket:message', {
+		detail: {
+			name: '',
+			data: {},
+			cached: false
+		}
+	}));
+}
+
+async function removeLabelButtonClickHandler(event: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	event.preventDefault();
+
+	const removeLabelButton = event.delegateTarget;
+
+	removeLabelButton.disabled = true;
+	await api.v3(`repos/${getRepoURL()}/issues/${getConversationNumber()!}/labels/${removeLabelButton.dataset.name!}`, {
+		method: 'DELETE'
+	});
+
+	updateSidebar();
+}
+
+function makeRemoveLabelButton(labelName: string, color: string, backgroundColor: string) {
+	const removeLabelButton = (
+		<button
+			type="button"
+			aria-label="Remove this label"
+			className="btn-link tooltipped tooltipped-nw rgh-remove-label-button"
+			data-name={labelName}
+		>
+			<XIcon/>
+		</button>
+	);
+
+	removeLabelButton.style.setProperty('--rgh-remove-label-button-bg-color', backgroundColor);
+	removeLabelButton.style.setProperty('--rgh-remove-label-button-color', color);
+
+	return removeLabelButton;
+}
+
+async function init(): Promise<void> {
+	await api.expectToken();
+
+	observe('.labels > a', {
+		constructor: HTMLElement,
+		add(label) {
+			// Override !important rule
+			label.style.setProperty('display', 'inline-flex', 'important');
```

Comment:
Probably: 
```suggestion
			label.classList.add('d-inline-flex', 'rgh-something');
```

Reply:
It doesn't work.
![Screenshot_20200813_023350](https://user-images.githubusercontent.com/36174850/90081325-bb05e980-dd15-11ea-9e05-4c24ee3cae9c.png)

Reply:
I don't like setting CSS via javascript when there's already a class and/or a CSS file we can use. You can add it via CSS with:

```css
:root .rgh-remove-label-button-already-added {
  display: inline-flex !important;
}
```

Maybe `:root` has to be doubled: `:root:root`. We use this format often, sometimes just to override the native selector without using `!important` (this is not the case)